src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -167,21 +167,149 @@ static u16 D_8082ABEC[] = {
-    9, 9, 9, 9, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-    0, 1, 0, 0, 1, 9, 0, 9, 0, 0, 9, 0, 0, 1, 1, 1, 0, 0, 0, 9, 9, 9, 1, 0, 0, 9, 9, 0,
+    {
+        AGE_REQ_ADULT, // Quiver/Bullet bag (Unused)
+        AGE_REQ_CHILD, // Kokiri sword
+        AGE_REQ_ADULT, // Master Sword
+        AGE_REQ_ADULT  // Big Goron's Sword
+    },
+    {
+        AGE_REQ_NONE,  // Bomb bag
+        AGE_REQ_CHILD, // Deku shield
+        AGE_REQ_NONE,  // Hylian shield
+        AGE_REQ_ADULT  // Mirror shield
+    },
+    {
+        AGE_REQ_ADULT, // Strength (Unused)
+        AGE_REQ_NONE,  // Kokiri tunic
+        AGE_REQ_ADULT, // Goron tunic
+        AGE_REQ_ADULT  // Zora tunic
+    },
+    {
+        AGE_REQ_NONE,  // Scale
+        AGE_REQ_NONE,  // Kokiri boots
+        AGE_REQ_ADULT, // Iron Boots
+        AGE_REQ_ADULT  // Hover Boots
+    },
+};
+
+// Controls if the icon is grayed out or not depending on age
+u8 gItemIconAgeInfo[] = {
+    AGE_REQ_CHILD, // Deku sticks
+    AGE_REQ_NONE,  // Deku nuts
+    AGE_REQ_NONE,  // Bombs
+    AGE_REQ_ADULT, // Bow
+    AGE_REQ_ADULT, // Fire arrows
+    AGE_REQ_NONE,  // Dins fire
+    AGE_REQ_CHILD, // Slingshot
+    AGE_REQ_NONE,  // Fairy Ocarina
+    AGE_REQ_NONE,  // Ocarina of Time
+    AGE_REQ_NONE,  // Bombchus
+    AGE_REQ_ADULT, // Hookshot
+    AGE_REQ_ADULT, // Longshot
+    AGE_REQ_ADULT, // Ice arrows
+    AGE_REQ_NONE,  // Farore's Wind
+    AGE_REQ_CHILD, // Boomerang
+    AGE_REQ_NONE,  // Lens of truth
+    AGE_REQ_CHILD, // Bean
+    AGE_REQ_ADULT, // Megaton Hammer
+    AGE_REQ_ADULT, // Light arrows
+    AGE_REQ_NONE,  // Nayrus love
+    AGE_REQ_NONE,  // Bottle
+    AGE_REQ_NONE,  // Red Potion
+    AGE_REQ_NONE,  // Green Potion
+    AGE_REQ_NONE,  // Blue Potion
+    AGE_REQ_NONE,  // Fairy
+    AGE_REQ_NONE,  // Fish
+    AGE_REQ_NONE,  // Milk
+    AGE_REQ_NONE,  // Letter in a bottle
+    AGE_REQ_NONE,  // Blue Potion
+    AGE_REQ_NONE,  // Bug
+    AGE_REQ_NONE,  // Big poe
+    AGE_REQ_NONE,  // Half Milk
+    AGE_REQ_NONE,  // Poe
+    AGE_REQ_CHILD, // Weird Egg
+    AGE_REQ_CHILD, // Chicken
+    AGE_REQ_CHILD, // Zelda's Letter
+    AGE_REQ_CHILD, // Keaton Mask
+    AGE_REQ_CHILD, // Skull Mask
+    AGE_REQ_CHILD, // Spooky Mask
+    AGE_REQ_CHILD, // Bunny Hood
+    AGE_REQ_CHILD, // Goron Mask
+    AGE_REQ_CHILD, // Zora Mask
+    AGE_REQ_CHILD, // Gerudo Mask
+    AGE_REQ_CHILD, // Mask of Truth
+    AGE_REQ_CHILD, // SOLD OUT
+    AGE_REQ_ADULT, // Pocket Egg
+    AGE_REQ_ADULT, // Pocket Cucco
+    AGE_REQ_ADULT, // Cojiro
+    AGE_REQ_ADULT, // Odd Mushroom
+    AGE_REQ_ADULT, // Odd Potion
+    AGE_REQ_ADULT, // Poacher's Saw
+    AGE_REQ_ADULT, // Goron's Sword (Broken)
+    AGE_REQ_ADULT, // Prescription
+    AGE_REQ_ADULT, // Eyeball Frog
+    AGE_REQ_ADULT, // Eye Drops
+    AGE_REQ_ADULT, // Claim Check
+    AGE_REQ_ADULT, // Unknown
+    AGE_REQ_ADULT, // Unknown
+    AGE_REQ_ADULT, // Unknown
```

Comment:
if this is indexed by `ItemID` then
```suggestion
    AGE_REQ_ADULT, // Bow with fire arrows
    AGE_REQ_ADULT, // Bow with ice arrows
    AGE_REQ_ADULT, // Bow with light arrows
```